src/server.js

```diff
@@ -42,6 +43,11 @@ module.exports = function() {
 		.set("view engine", "html")
 		.set("views", path.join(__dirname, "..", "client"));
 
+	app.get("/plugins/themes/:theme.css", (req, res) => {
+		const theme = req.params.theme;
+		return res.sendFile(themes.fileName(theme));
```

Comment:
Can't set it project-wide (as in, effect other users), but locally you can do `npm config set save-exact true`